cypress/e2e/basic.cy.js

```diff
@@ -1,19 +1,51 @@
-const testURLRoot = 'http://127.0.0.1:50722/templates/'
-
-describe('Basic site functionality', () => {
-
-  it('displays the under development notice', () => {
-    cy.intercept({
-      method: 'GET',
-      url: '/template',
-      hostname: 'api-test.ramanchada.ideaconsult.net',
-    }, {
-      fixture: 'json/bk_rcapi_template_default.json',
-    })
-    cy.visit(testURLRoot)
+const testURLRoot = "http://localhost:5173/templates";
+
+describe("Basic site functionality", () => {
+  beforeEach(() => {
+    cy.intercept(
+      {
+        method: "GET",
+        url: "/template",
+        hostname: "api-test.ramanchada.ideaconsult.net",
+      },
+      {
+        fixture: "json/bk_rcapi_template_default.json",
+      }
+    ).as("getAllTemplates");
```

Comment:
Why do we need this when we still define the intercepts in the tests themselves?